server/views/forms/agency_notification.pug

```diff
@@ -26,6 +26,8 @@ block content
     tr.top.spaceUnder
       td Gender:
 
+  p Is the prisoner partaking in the Acquisitive Crime project with a mandatory trail monitoring<br>condition? Yes No
```

Comment:
the br makes it look a bit weird but see your conundrum regarding length of question and wrapping. Worth getting this out and showing Louise I think